build-logic/cleanup/src/main/kotlin/gradlebuild/cleanup/services/DaemonTracker.kt

```diff
@@ -1,113 +0,0 @@
-/*
- * Copyright 2019 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package gradlebuild.cleanup.services
-
-import org.gradle.api.file.DirectoryProperty
-import org.gradle.api.services.BuildService
-import org.gradle.api.services.BuildServiceParameters
-import org.gradle.api.tasks.testing.TestDescriptor
-import org.gradle.api.tasks.testing.TestListener
-import org.gradle.api.tasks.testing.TestResult
-import org.gradle.process.ExecOperations
-import org.slf4j.LoggerFactory
-import java.util.concurrent.ConcurrentHashMap
-import javax.inject.Inject
-
-
-private
-val logger = LoggerFactory.getLogger("daemonTracker")
-
-
-abstract class DaemonTracker : BuildService<DaemonTracker.Params>, AutoCloseable {
```

Comment:
I think we still should kill the Java processes started from tests when running any Gradle build. This is especially important for builds on developer machines.